ymer/src/main/java/com/avanza/ymer/RepeatingTask.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015 Avanza Bank AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.avanza.ymer;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+import java.time.Duration;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+
+class RepeatingTask implements AutoCloseable{
+	private final ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
```

Comment:
I suggest to use deamon threads with a descriptive name here

Reply:
Fixed in https://github.com/AvanzaBank/ymer/pull/74/commits/b87f6421851adf2d546ad9ec31d91bfc3ea4dd0a

Reply:
Looks good to me! 👍